mmdeploy/backend/coreml/torchscript2coreml.py

```diff
@@ -47,30 +50,22 @@ def create_shape(name: str, input_shapes: Dict) -> ct.Shape:
     return ct.TensorType(shape=shape, name=name)
 
 
-def from_torchscript(torchscript_model: Union[str,
+def from_torchscript(model_id: int,
+                     torchscript_model: Union[str,
                                               torch.jit.RecursiveScriptModule],
-                     output_file_prefix: str,
-                     input_names: Sequence[str],
-                     output_names: Sequence[str],
-                     input_shapes: Dict,
-                     convert_to: str = 'neuralnetwork',
-                     fp16_mode: bool = False,
-                     skip_model_load: bool = True,
-                     **kwargs):
+                     output_file_prefix: str, deploy_cfg: Union[str,
```

Comment:
Did we decide NOT accept deploy_cfg as the input?

Reply:
My mistake. PR #1308 will not be merged to dev-1.x until it is fixed in master branch